Bottom: ba243ac6fb44dcff3dfdd31f413d8ed7517dbd5a
Top:    310b3de037a682a6de155f870cee9b09e4bad56f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-21 21:21:37 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
